src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -501,7 +512,7 @@ public override bool VisitNamedType(INamedTypeSymbol symbol)
             }
         }
 
-        private class ReferenceGenerator : SymbolVisitor<bool>
+        private sealed class ReferenceGenerator : SymbolVisitor
```

Comment:
the bool in this type was only used to determine if any text was written when hitting a container (in order ot determine if a '.' was needed between nested types/namespaces).  We don't need that (and the bool could be confused for success/failure), so removed.